views/containers/Camera.jsx

```diff
@@ -9,6 +9,10 @@ const ReactTHREE = require('react-three');
 const PerspectiveCamera = ReactTHREE.PerspectiveCamera;
 const keys = [KEYS.J, KEYS.K, KEYS.L, KEYS.I, KEYS.ENTER];
 
+const rotateCamera = ({ cameraRotation }, event) => {
+  cameraRotation(event);
+};
```

Comment:
I don't understand why you do this here, just dispatch the action `onKeyPressed`